### PR TITLE
add angular-img-cropper@1.1.0 w/ git auto-update

### LINKED ISSUE #8207: [Request] Add angular-img-cropper
**Library name:** angular-img-cropper
**Git repository url:** https://github.com/AllanBishop/angular-img-cropper
**npm package url(optional):** 
**License(s):** MIT
**Official homepage:** https://github.com/AllanBishop/angular-img-cropper
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #8207 
@x09326 Please help me review this pull request, thank you.
use git auto-update instead of npm auto-update, because author didn't reply.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/AllanBishop/angular-img-cropper
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 129
   - Count of watchers: 15
   - Count of forks: 90
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8207, cc @AllanBishop

### CODE REVIEW

File: ajax/libs/angular-img-cropper/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "angular-img-cropper",
+  "filename": "",
```
Comment (x09326) [inline ajax/libs/angular-img-cropper/package.json]: @kennynaoh 
Please add the filename.

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by PeterDaveHello** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@kennynaoh why to squash? We should use merge here.

**Comment by kennynaoh** [comment]
OK

**Comment by PeterDaveHello** [comment]
@kennynaoh What do you mean "OK"?

**Comment by kennynaoh** [comment]
I will just use merge here.

**Comment by PeterDaveHello** [comment]
@kennynaoh but you DID use "SQUASH and merge" feature, any reason?

**Comment by kennynaoh** [comment]
I misremember, sorry about it. I will be more careful next time. Thanks!

**Pull request merged**